_posts/en/newsletters/2024-12-06-newsletter.md

```diff
@@ -0,0 +1,86 @@
+---
+title: 'Bitcoin Optech Newsletter #332'
+permalink: /en/newsletters/2024/12/06/
+name: 2024-12-06-newsletter
+slug: 2024-12-06-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Continued discussion about consensus cleanup soft fork proposal:**
+  Antoine Poinsot [posted][poinsot time warp] to the existing Delving
+  Bitcoin thread about the [consensus cleanup][topic consensus cleanup]
+  soft fork proposal.  In addition to the already proposed fix for the
+  classic [time warp vulnerability][topic time warp], he proposed also
+  including a fix for the recently discovered Zawy-Murch time warp (see
+  [Newsletter #316][news316 time warp]).  He favored a fix originally
+  proposed by Mark "Murch" Erhardt, "require that the last block in a
+  difficulty period _N_ has a higher timestamp than the first block in
+  the same difficulty period _N_".
+
+  Anthony Towns [favored][towns time warp] an alternative solution
+  proposed by Zawy that would forbid any block from claiming it was
+  produced more than two hours before any previous block.  Zawy
+  [noted][zawy time warp] that his every-block solution increased the
+  risk of miners losing money from running outdated software but it
+  would also make timestamps more accurate for other uses, such as time
+  locks (e.g., [HTLCs][topic htlc]).
+
+  Separately, both on [Delving Bitcoin][poinsot duptx delv] and the
+  [Bitcoin-Dev mailing list][poinsot duptx ml], Poinsot sought feedback
+  about which proposed solution to use to prevent block 1,983,702 and
+  some later blocks from [duplicating][topic duplicate transactions] a
+  previous coinbase transaction (which could lead to loss of money and
+  the creation of an attack vector).  Four proposed solutions are
+  presented, all of which would only directly affect miners---so
+  feedback from miners would be especially appreciated.
```

Comment:
Could link "feedback from miners" to https://groups.google.com/g/bitcoinminingdev/c/qyrPzU1WKSI